node/src/chain_spec.rs

```diff
@@ -181,3 +239,40 @@ fn testnet_genesis(
 		}),
 	}
 }
+
+// testnet config
+pub fn ibtida_config() -> Result<ChainSpec, String> {
```

Comment:
should not boot from here

Reply:
Probably worth just removing. No problem keeping the JSON chainspec, right?

Reply:
@justinFrevert ?